src/groovy/org/pih/warehouse/api/StockMovementItem.groovy

```diff
@@ -301,6 +301,10 @@ class AvailableItem {
         quantityAvailable(nullable: true)
     }
 
+    def getIsAutopickable() {
```

Comment:
I think my only beef is with the camelcase so let's go with isAutoPickable to make it more readable.